src/moments.jl

```diff
@@ -58,99 +78,179 @@ function var!(R::AbstractArray, A::RealArray, wv::WeightVec, dim::Int; mean=noth
+             mean=mean, corrected=corrected)
     end
+end
 
 ## std
 """
-    stdm(v, wv::WeightVec, m, [dim])
-
-Return the standard deviation of a real-valued array `v` with a known mean `m`,
-optionally over a dimension `dim`. The weighting vector `wv` specifies frequency
-weights (also called case weights) for the estimate.
+    stdm(v, w::AbstractWeights, m, [dim]; corrected=false)
+
+Compute the standard deviation of a real-valued array `x` with a known mean `m`,
+optionally over a dimension `dim`. Observations in `x` are weighted using weight vector `w`.
+The uncorrected (when `corrected=false`) sample standard deviation is defined as:
+```math
+\\sqrt{\\frac{1}{\\sum{w}} \\sum_{i=1}^n {w_i\\left({x_i - m}\\right)^2 }}
+```
+where ``n`` is the length of the input. The unbiased estimate (when `corrected=true`) of the
+population standard deviation is computed by replacing ``\\frac{1}{\\sum{w}}`` with a factor
+dependent on the type of weights used:
+* `AnalyticWeights`: ``\\frac{1}{\\sum w - \\sum {w^2} / \\sum w}``
+* `FrequencyWeights`: ``\\frac{1}{\\sum{w} - 1}``
+* `ProbabilityWeights`: ``\\frac{n}{(n - 1) \\sum w}`` where ``n`` equals `count(!iszero, w)`
+* `Weights`: `ArgumentError` (bias correction not supported)
 """
-Base.stdm(v::RealArray, wv::WeightVec, m::Real) = sqrt(varm(v, wv, m))
+Base.stdm(v::RealArray, w::AbstractWeights, m::Real; corrected::DepBool=nothing) =
+    sqrt(varm(v, w, m, corrected=depcheck(:stdm, corrected)))
 
 """
-    std(v, wv::WeightVec, [dim]; mean=nothing)
-
-Return the standard deviation of a real-valued array `v`, optionally over a
-dimension `dim`. The weighting vector `wv` specifies frequency weights (also
-called case weights) for the estimate.
+    std(v, w::AbstractWeights, [dim]; mean=nothing, corrected=false)
+
+Compute the standard deviation of a real-valued array `x`,
+optionally over a dimension `dim`. Observations in `x` are weighted using weight vector `w`.
+The uncorrected (when `corrected=false`) sample standard deviation is defined as:
+```math
+\\sqrt{\\frac{1}{\\sum{w}} \\sum_{i=1}^n {w_i\\left({x_i - μ}\\right)^2 }}
+```
+where ``n`` is the length of the input and ``μ`` is the mean.
+The unbiased estimate (when `corrected=true`) of the population standard deviation is
+computed by replacing ``\\frac{1}{\\sum{w}}`` with a factor dependent on the type of
+weights used:
+* `AnalyticWeights`: ``\\frac{1}{\\sum w - \\sum {w^2} / \\sum w}``
+* `FrequencyWeights`: ``\\frac{1}{\\sum{w} - 1}``
+* `ProbabilityWeights`: ``\\frac{n}{(n - 1) \\sum w}`` where ``n`` equals `count(!iszero, w)`
+* `Weights`: `ArgumentError` (bias correction not supported)
 """
-Base.std(v::RealArray, wv::WeightVec; mean=nothing) = sqrt.(var(v, wv; mean=mean))
+Base.std(v::RealArray, w::AbstractWeights; mean=nothing, corrected::DepBool=nothing) =
+    sqrt.(var(v, w; mean=mean, corrected=depcheck(:std, corrected)))
+
+Base.stdm(v::RealArray, m::RealArray, dim::Int; corrected::DepBool=nothing) =
```

Comment:
Yeah, that's been there for a while, but I didn't notice when I updated it. We had a discussion on #248 about it. I think the plan was to move this method to base julia and add a version check, but I haven't gotten around to doing it yet.